x-pack/solutions/security/plugins/security_solution/server/endpoint/services/workflow_insights/builders/index.ts

```diff
@@ -25,9 +26,12 @@ export interface BuildWorkflowInsightParams {
 export function buildWorkflowInsights(
   params: BuildWorkflowInsightParams
 ): Promise<SecurityWorkflowInsight[]> {
-  if (params.request.body.insightType === DefendInsightType.Enum.incompatible_antivirus) {
-    return buildIncompatibleAntivirusWorkflowInsights(params);
+  switch (params.request.body.insightType) {
+    case DefendInsightType.Enum.incompatible_antivirus:
+      return buildIncompatibleAntivirusWorkflowInsights(params);
+    case DefendInsightType.Enum.policy_response_failure:
+      return buildPolicyResponseFailureWorkflowInsights(params);
```

Comment:
Should we put it behind a feature flag?

Reply:
This is flagged at the [API level](https://github.com/joeypoon/kibana/blob/a34117faab9f3a46be28d0f44f52bdf75f30222a/x-pack/solutions/security/plugins/elastic_assistant/server/routes/defend_insights/post_defend_insights.ts#L135). Agree that we'll want a flag at the UI level as well but that will be in separate PR when we add the API call for this insight type.